cmd/osctl/cmd/list.go

```diff
@@ -90,10 +97,31 @@ var lsCmd = &cobra.Command{
 					if info.Link != "" {
 						display += " -> " + info.Link
 					}
-					fmt.Fprintf(w, "%s\t%d\t%s\t%s\n",
+
+					var size string
+					if humanizeFlag {
+						size = humanize.Bytes(uint64(info.Size))
+					} else {
+						size = fmt.Sprintf("%d", info.Size)
```

Comment:
nit: we can avoid the `else` if we assign: `size := fmt.Sprintf("%d", info.Size)`.

Reply:
yep, I was torn between inefficient assignment (in case of humanizeFlag) and extra `else` ;)

let's go better looking code route.

Reply:
I could go either way.